translations/en.json

```diff
@@ -1032,7 +1032,7 @@
     "migrations-failed-title": "Migration failed",
     "migrations-failed-content": "{{message}}\nschema version: initial {{initial-version}}, current {{current-version}}, last {{last-version}}\n\nPlease let us know about this problem at #status public chat. If you press \"Cancel\" button, nothing will happen. If you press \"{{erase-multiaccounts-data-button-text}}\" button, multiaccount's db will be removed and you will be able to unlock multiaccount. All multiaccount's data will be lost.",
     "migrations-erase-multiaccounts-data-button": "Erase multiaccount's db",
-    "multiaccount-and-db-password-mismatch-title": "The problem occurred!",
```

Comment:
THE problem?! THE problem to rule ALL problems?!
![pasta la vista](https://user-images.githubusercontent.com/2212681/62298294-c1922d80-b440-11e9-93e9-dccaf937b90a.jpg)